deployment/arm/deploy-region.sh

```diff
@@ -71,21 +72,29 @@ echo "Environment: $env"
 echo "Region: $region"
 echo
 
-echo "Creating Regional Resource Group: $resourceGroupName"
+echo "######## Creating Regional Resource Group: $resourceGroupName ########"
 az group create \
     --name $resourceGroupName \
     --location $region
+
+echo 
+SLEEP_DURATION=10
```

Comment:
how did we come up with the 10 seconds? is this an SLA for RG creation? My concern is that for you it required 10 and it worked fine for Wallace (0 seconds), we need to make sure whatever value we use here would work overall. It would also be nice to add a note about this in the deployment md file in case someone else runs into it

Reply:
There's a comment right below this explaining why the sleep is needed.  Also, not sure if anyone will be running their own deployment and not using the script.

Reply:
@mydiemho On friday when i was re-creating environments multiple times I wasn't able to repro this happening via the pipelines.  Could this be some isolated environment setup?